### DISCUSSION TITLE
streams-memory_raw_i2s  problems

### DISCUSSION THREAD

**Discussion by jialarcon** [pschatzmann/arduino-audio-tools#281] [Q&A] [answered]
Hi, I have successfully run your example with StarWars30.h but when I try to use a raw file of my own the sound always is played back very fast. I am exporting from audacity (only one channel) at 20050 / 16 bit as raw and after using HX to set it as a C file.

It looks I am missing something, any clues ?

Kind Regards

Jose Alarcon

**Comment by pschatzmann** [comment, marked as answer]
I would expect that the 
```
cfg.channels=1;
cfg.sample_rate= 20050; 
```
has not been set correctly in your program.

**Reply by jialarcon** [reply]
Thank you for your comment Phil, you are right, after changing channels from 2 to 1 it worked. 

I based my sketch on your Starwars30_raw example which I suppose was deriived from a 2 channel audio file.

Regards
